### PR TITLE
Update global_data_MOD.js

### PR DESCRIPTION
**Pull request by Phyrok**
**Please describe the changes this PR makes and why it should be merged:**
Outdated looks updated to look more appealing with the rest of the actions.

Removal of unnecessary code in **Check_Global_Data.js** as it DBM has a build in conditional input. 

| **Check_Global_Data.js** | **Control_Global_Data.js** | **Delete_Global_Data.js** | **Store_Global_Data.js** |
| :--- | :--- | :--- | :--- | 
| <details> <summary>Before</summary> image </details> | <details> <summary>Before</summary> image </details> | <details> <summary>Before</summary> image </details> | <details> <summary>Before</summary> image </details>
|<details> <summary>After</summary> image </details> | <details> <summary>After</summary> image </details> | <details> <summary>After</summary> image </details> | <details> <summary>After</summary> image </details>

**Status**

- [x] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes
- [ ] Documentation has been added/modified, or there is nothing to change (docs/mods.json)

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by Phyrok** [comment]
I think I closed the previous by removing the repo of my GitHub Desktop.
Sorry for the trouble I am causing.

File: actions/check_global_data_MOD.js [hunk 1]
```diff
@@ -35,76 +34,33 @@ module.exports = {
...
+<conditional-input id="branch" style="padding-top: 16px;"></conditional-input>`;
+  },
+
+  init() {
+    const { glob } = this;
+    glob.onComparisonChanged = function onComparisonChanged(event) {
+      if (event.value === '0') {
+        getElementById('directValue').style.display = 'none';
+      } else {
+        getElementById('directValue').style.display = null;
```
Comment (TheMonDon) [inline actions/check_global_data_MOD.js:59]:
```suggestion
        document.getElementById('directValue').style.display = null;
```

File: actions/check_global_data_MOD.js [hunk 2]
```diff
@@ -35,76 +34,33 @@ module.exports = {
...
-        default:
-          break;
+<conditional-input id="branch" style="padding-top: 16px;"></conditional-input>`;
+  },
+
+  init() {
+    const { glob } = this;
+    glob.onComparisonChanged = function onComparisonChanged(event) {
+      if (event.value === '0') {
+        getElementById('directValue').style.display = 'none';
```
Comment (TheMonDon) [inline actions/check_global_data_MOD.js:57]:
```suggestion
        document.getElementById('directValue').style.display = 'none';
```

File: actions/check_global_data_MOD.js [hunk 3]
```diff
@@ -35,76 +34,33 @@ module.exports = {
...
-          document.getElementById('iffalseName').innerHTML = 'Anchor ID';
-          document.getElementById('iffalseContainer').style.display = null;
-          break;
-        default:
-          break;
+<conditional-input id="branch" style="padding-top: 16px;"></conditional-input>`;
+  },
+
+  init() {
+    const { glob } = this;
```
Comment (TheMonDon) [inline actions/check_global_data_MOD.js:54]:
```suggestion
    const { glob, document } = this;
```

**Review by TheMonDon — CHANGES_REQUESTED** [review]
My bad I told you to remove document not what i meant

File: actions/check_global_data_MOD.js [hunk 4]
```diff
@@ -35,76 +34,33 @@ module.exports = {
...
-          document.getElementById('iffalseContainer').style.display = null;
-          break;
-        default:
-          break;
-      }
-    };
-    glob.onChangeTrue(document.getElementById('iftrue'));
-    glob.onChangeFalse(document.getElementById('iffalse'));
+
+    glob.onComparisonChanged(getElementById('comparison'));
```
Comment (TheMonDon) [inline actions/check_global_data_MOD.js:63]:
```suggestion
    document.glob.onComparisonChanged(getElementById('comparison'));
```

**Review by TheMonDon — CHANGES_REQUESTED** [review]

File: actions/check_global_data_MOD.js [hunk 5]
```diff
@@ -35,76 +34,33 @@ module.exports = {
...
-          document.getElementById('iffalseContainer').style.display = null;
-          break;
-        default:
-          break;
-      }
-    };
-    glob.onChangeTrue(document.getElementById('iftrue'));
-    glob.onChangeFalse(document.getElementById('iffalse'));
+
+    document.glob.onComparisonChanged(getElementById('comparison'));
```
Comment (TheMonDon) [inline actions/check_global_data_MOD.js:63]:
```suggestion
    glob.onComparisonChanged(document.getElementById('comparison'));
```

**Review by TheMonDon — CHANGES_REQUESTED** [review]

File: actions/check_global_data_MOD.js [hunk 6]
```diff
@@ -8,24 +8,23 @@ module.exports = {
     authorUrl: 'https://github.com/dbm-network/mods',
     downloadURL: 'https://github.com/dbm-network/mods/blob/master/actions/check_global_data_MOD.js',
   },
-
   subtitle(data) {
```
Comment (TheMonDon) [inline actions/check_global_data_MOD.js:11]:
```suggestion

  subtitle(data) {
```

File: actions/control_global_data_MOD.js [hunk 7]
```diff
@@ -8,7 +8,6 @@ module.exports = {
     authorUrl: 'https://github.com/dbm-network/mods',
     downloadURL: 'https://github.com/dbm-network/mods/blob/master/actions/control_global_data_MOD.js',
   },
-
   subtitle(data) {
```
Comment (TheMonDon) [inline actions/control_global_data_MOD.js:11]:
```suggestion

  subtitle(data) {
```

File: actions/delete_global_data_MOD.js [hunk 8]
```diff
@@ -20,7 +18,7 @@ module.exports = {
     return `
 <div style="padding-top: 8px;">
   <div style="float: left; width: 80%;">
-    Data Name:<br>
+    <span class="dbminputlabel">Data Name</span><br>
     <input id="dataName" class="round" placeholder="Leave it blank to delete all data" type="text">
```
Comment (TheMonDon) [inline actions/delete_global_data_MOD.js:22]:
```suggestion
    <input id="dataName" class="round" placeholder="Leave blank to delete all data" type="text">
```

File: actions/delete_global_data_MOD.js [hunk 9]
```diff
@@ -9,7 +8,6 @@ module.exports = {
     authorUrl: 'https://github.com/dbm-network/mods',
     downloadURL: 'https://github.com/dbm-network/mods/blob/master/actions/delete_global_data_MOD.js',
   },
-
   subtitle(data) {
```
Comment (TheMonDon) [inline actions/delete_global_data_MOD.js:11]:
```suggestion

  subtitle(data) {
```

File: actions/store_global_data_MOD.js [hunk 10]
```diff
@@ -8,11 +8,9 @@ module.exports = {
     authorUrl: 'https://github.com/dbm-network/mods',
     downloadURL: 'https://github.com/dbm-network/mods/blob/master/actions/store_global_data_MOD.js',
   },
-
   subtitle(data, presets) {
```
Comment (TheMonDon) [inline actions/store_global_data_MOD.js:11]:
```suggestion

  subtitle(data, presets) {
```

File: actions/store_global_data_MOD.js [hunk 11]
```diff
@@ -8,11 +8,9 @@ module.exports = {
...
     return presets.getVariableText(data.storage, data.varName);
   },
-
   variableStorage(data, varType) {
```
Comment (TheMonDon) [inline actions/store_global_data_MOD.js:14]:
```suggestion

  variableStorage(data, varType) {
```

**Review by TheMonDon — CHANGES_REQUESTED** [review]

File: actions/store_global_data_MOD.js (hunk 11 above)
Comment (TheMonDon) [reply actions/store_global_data_MOD.js:14]: I think this is all the changes I'd like, i'll take a look after these are added and then approve the PR.

**Review by TheMonDon — APPROVED** [review]

**Pull request merged**